EnglishTranslation-master/SKILL.tsv

```diff
@@ -2102,7 +2102,7 @@ SKILL_20150317_002101	Decreases damage received by 10% per level when at least 3
 SKILL_20150317_002102	Two-handed Sword Mastery	투핸드소드 마스터리				
 SKILL_20150317_002103	[Two-handed Sword] wearing critical attack is increased by 10% per level of features.	[양손검] 착용 시 치명타 공격력이 특성 레벨 당 10% 증가합니다.				
 SKILL_20150317_002104	Moulinet: Critical Rate	뮬리넷: 치명타발생				
-SKILL_20150317_002105	Critical rate for [Moulinet] is increased by 10% per Attribute level.	스킬 [뮬리넷]의 치명타 확률이 특성 레벨 당 10% 증가합니다.				
+SKILL_20150317_002105	Critical Rate for [Moulinet] is increased by 10% per Attribute level.	스킬 [뮬리넷]의 치명타 확률이 특성 레벨 당 10% 증가합니다.				
```

Comment:
See above comment about the standardization. Also 'Attribute' should be lowercase. 

Take a look at these attributes: https://tos.neet.tv/skills/10001